biz_portal/apps/portal/templates/portal/business_list.html

```diff
@@ -1,8 +1,89 @@
+{% extends "base.html" %}
 {% load param_replace %}
+{% load static %}
 
-{% include 'portal/search_snippet.html' %}
+<!-- {% include 'portal/search_snippet.html' %} -->
```

Comment:
we don't keep commented out code. Also django will still insert this snippet even if it's in an HTML comment. If you want to exclude a django tag, use `{# .... #}` I think

Reply:
Covered